pkg/epp/common/config/defaults.go

```diff
@@ -18,11 +18,18 @@ limitations under the License.
 // different EPP components.
 package config
 
+import "time"
+
 const (
 	// DefaultKVCacheThreshold is the default KV cache utilization (0.0 to 1.0)
 	// threshold.
 	DefaultKVCacheThreshold = 0.8
 	// DefaultQueueThresholdCritical is the default backend waiting queue size
 	// threshold.
 	DefaultQueueThresholdCritical = 5
+	// DefaultMetricsStalenessThreshold defines how old metrics can be before they
+	// are considered stale.
+	// Given the pod metrics refresh interval is 50ms, a threshold slightly above
+	// that should be fine.
+	DefaultMetricsStalenessThreshold = 200 * time.Millisecond
```

Comment:
The staleness is determined by the refresh internal plus the latency of the metrics API. The metrics API is generally super fast but we haven't done a thorough analysis on worst case scenarios.

To be on the safer side, I suggest we start with a larger threshold such as 2s.